wp_api/src/login.rs

```diff
@@ -332,6 +363,39 @@ mod tests {
             .is_empty());
     }
 
+    #[rstest]
+    #[case("api-details/invalid-json-01.json")]
+    #[case("api-details/invalid-json-02.json")]
+    fn test_valid_json_documents(#[case] input: &str) {
```

Comment:
The test files are called `invalid-json-..` whereas the test is called `test_valid_json_documents`. The assertion seems to expect the files to be valid, so could we rename the files to match the expectation?

Reply:
Addressed in https://github.com/Automattic/wordpress-rs/pull/598/commits/a3e84220c137e41bb5ae6d38c18ad0f5c1fda868